umh.docs.umh.app/content/en/docs/datacontracts/_index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Data Contracts / API"
+menuTitle: "Data Contracts / API"
+description: "This page describes how messages flow in the UMH, which message goes where, how it has to be formatted and how you can create your own structures."
+weight: 1725
+---
+
+## What are Data Contracts
+
+Data Contracts are formal agreements that define the structure, format, and
+rules for data exchanged between components within a Unified Namespace (UNS)
+architecture. They specify metadata, data models, and service levels to ensure
+that all systems interact consistently and reliably. By establishing clear
+guidelines for data exchange, Data Contracts facilitate seamless integration
+and prevent disruptions caused by changes in data formats or structures.
+
+In short, they define where a message is going, what structure it must follow,
+how it gets there, and what happens when it arrives, based on defined rules and
+services.
+
+### Example Historian
+
+To give you a simple example, just think about the `_historian' schema. Perhaps
+without realizing it, you have already used the Historian Data Contract by using
+this schema. The Historian Data Contract governs the flow of data from your
+machines to the database by enforcing rules and providing services:
+
+- **Protocol Converter**: You must use a protocol converter to send your data
+from your machine to your UNS.
+- **Topic structure**: You must use the correct topic structure and include the
+`_historian` schema.
+- **Message format**: The payload must be JSON and follow the correct message
+format, such as including a timestamp.
+- **Data Bridges**: The Historian Data Contract includes two Data Bridges that
+move your data from MQTT to Kafka and vice versa.
+- **Kafka to Postgres**: This is a special microservice of UMH that allows
+Kafka to write data to the database.
+
+These rules ensure that the data can be written to the database without causing
+errors, and that the data can be read by other programs that know what data to
+expect. The Data Bridges enable seamless data flow via MQTT without any
+additional configuration.
+
+{{% notice note %}}
+If you want a lot more information and really dive into the reasons for this
+approach, we recommend our article about
+[Data Modeling in the UNS](https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/)
+on our Learn page.
+<!-- Add link to Jeremys upcoming article, once it is released. -->
+{{% /notice %}}
+
+## Components of a Data Contract
+
+Data Contracts have several components and rules. This section provides a
```

Comment:
It feels weird that "topic structure" is in the same heading level like "protocol converter". Maybe split it up into formal and implementation?

### Formal Components

#### Topic Structure

#### Payload Structure

### Implementation

#### Protocol Converters

...